release_docs/RELEASE.txt

```diff
@@ -615,6 +615,9 @@ Platforms Tested
 Known Problems
 ==============
 
+    Building HDF5 Fortran on windows with Intel oneAPI 2023.2 currently fails 
+    with link errors.  Windows binaries will be without Fortran for this release.
+ 
```

Comment:
Building HDF5 Fortran on Windows with Intel oneAPI 2023.2 currently fails with link errors. As a result, this release of Windows binaries will not include Fortran.